src/components/CvTable/CvList.js

```diff
@@ -0,0 +1,148 @@
+import React, { useEffect, useState } from "react";
+import * as numeral from "numeral";
+import { CvInformation } from "./CvInformation";
+import { CvTable } from "../ReusableComponents/CvTable";
+import { connect } from "react-redux";
+import { setFavoriteList } from "../../actions/cvActions";
+import { EditFavoriteListButton } from "../ReusableComponents/Buttons";
+
+function CvList({
+  rawList,
+  fetchCvList,
+  loading,
+  setFavorites,
+  favorites,
+  favoriteCvList,
+  setFavoriteList
+}) {
+  const [cvList, setCvList] = useState([]);
+  const [filteredInfo, setFilteredInfo] = useState({});
+  /*
+  todo use in request to api
+  const [sortedInfo, setSortedInfo] = useState({});
+  */
+  const [salaryFilterRange, setSalaryFilterRange] = useState([]);
+  const [salaryRange, setSalaryRange] = useState([]);
+  const [cvInfo, setCvInfo] = useState({ visible: false, cvInformation: {} });
+
+  useEffect(() => {
+    (async function() {
+      let cvList = rawList.length > 0 && rawList;
+      if (!cvList) {
+        cvList = await fetchCvList();
+      }
+      setCvList(cvList);
+      if (rawList && rawList.length) {
```

Comment:
remove condition

Reply:
i changed rawList for cvList and i can`t remove condition

Reply:
this part of code was change in other branch(work_with_favorites) according to comment